### ISSUE TITLE
[Quest] Audience with the Prophet

### ISSUE THREAD

**Issue by phikaro** [HellscreamWoW/Tracker#2470]
The bug with this quest is not showing it on the map in Exodar where to turn it in.

http://www.wowhead.com/quest=9698/audience-with-the-prophet

**Comment by Malae1** [comment]
Is Velen in the Vault of Lights in Exodar(past the holograms and up the stairs)?

**Comment by phikaro** [comment]
Yes it's there in Exodar, I found him as I know his location from before, but still I think it's a bug not to show him on the map.
